### ISSUE TITLE
Clarify location for config file

### ISSUE THREAD

**Issue by erithmetic** [ashwanthkumar/gocd-slack-build-notifier#1]
I have installed the plugin and it seems to be installed without any errors. However, I'm not seeing any notifications in slack.

There is an instruction:

> Create a file go_notify.conf in the server's home directory

I'm not sure exactly what the server's home directory is and I'm wondering if this is my problem. I created a conf file in `/var/go` and `/var/lib/go-server` on a machine running ubuntu with an ansible-installed go.cd.

**Comment by ashwanthkumar** [comment]
I'm sorry for the instruction is not clear. You need `go_notify.conf` on home directory of the Go Server. Since Go doesn't have support for global configuration mechanism, the plugin reads the file from the user's home directory.

**Comment by ashwanthkumar** [comment]
Did it help? If you still have issues can you share the snippet of the `plugin-slack.notifier.log`?

**Comment by erithmetic** [comment]
Thanks for checking in, but unfortunately, no, I haven't gotten it to work. I don't see that log file anywhere. I have `go_notify.conf` in several locations, owned by the `go` user (the user that the server and agents run as:
- `/var/go/go_notify.conf` - this is the go user's home directory
- `/var/lib/go-server/go_notify.conf` - this is the directory that holds the server jars, plugins, etc.
- `/var/lib/go-agent1/go_notify.conf` - this is the directory that holds the agents' jars, pipelines, plugins, etc.

I have the notifier in `/var/lib/go-agent1/plugins/external` and in `/var/lib/go-server/plugins/external`. The Go.CD admin page shows that the plugin is installed. I don't see anything in the logs that indicates an error.

Here are the contents of my config:

```
gocd.slack {
  server-host = "https://my.domain.com/"
  webhookUrl = "https://hooks.slack.com/services/SOMERANDOMHASH"
}
```

**Comment by ashwanthkumar** [comment]
Just so we're all on the same page. What version of GoCD are you using? The official release that supports notification plugin is yet to be released (version 15.1). If you're on a older version it will not work.

**Comment by erithmetic** [comment]
Aha! I've been using v14.4. Thanks for your help!
